src/renderer/helpers/api/local.js

```diff
@@ -368,11 +372,16 @@ export async function getLocalChannelId(url) {
       } else if (navigationEndpoint.metadata.page_type === 'WEB_PAGE_TYPE_UNKNOWN' && navigationEndpoint.payload.url?.startsWith('https://www.youtube.com/')) {
         // handle redirects like https://www.youtube.com/@wanderbots, which resolves to https://www.youtube.com/Wanderbots, which we need to resolve again
         url = navigationEndpoint.payload.url
-      } else {
-        return null
+      } else if (navigationEndpoint.payload.browseId === 'FEpost_detail') {
+        // convert base64 params to string and get the channelid (if this gets updated, we should look into using a protobuf library instead)
+        return atob(navigationEndpoint.payload.params).replaceAll(/[^\d\sA-Za-z-]/g, ' ').trim().split(' ').at(-1)
```

Comment:
I believe this is the only thing that changes (and adding doLogError) since the last review.

Reply:
Replying to the code comment: We don't need to use a protobuf library, YouTube.js already has the protobuf, all you would need to do is add a decode function to YouTube.js's ProtoUtils file and then call that function here.

Reply:
Interesting, I wasn't aware of that file. I was thinking of using `@bufbuild/protobuf` which is the same library that YouTube.js uses but that would work better